src/amber/pipes/static.cr

```diff
@@ -43,6 +41,11 @@ module Amber
         call_next_with_file_path(context, request_path, file_path)
       end
 
+      private def get_current_scope(context)
+        scope = context.request.matched_route.payload.try { |r| r.scope.to_s }
+        scope ? scope : ""
+      end
```

Comment:
Can be simplified by `context.request.matched_route.payload.try { |r| r.scope.to_s } || ""`

Reply:
Thank you, I changed it.